README.md

```diff
@@ -253,13 +259,13 @@ export AGENT1=104.248.137.25
 * Install the first server
 
 ```bash
-k3sup install --user root --ip $SERVER1 --datastore="${DATASTORE}"
+k3sup install --user root --ip $SERVER1 --datastore="${DATASTORE}" --token=${TOKEN}
```

Comment:
Part of me thinks that this should be handled by K3sup rather than having the user collect this value.

Or are you saying that the workflow would be to create the token on local environment and pass it in for calls?